scripts/test_translation_templates.py

```diff
@@ -0,0 +1,132 @@
+from colors import color
+
+import os
+import shutil
+import docker
+import sys
+import subprocess
+
+SCHEMAS_TO_EXTRACT = ["census_individual_gb_wls", "census_individual_gb_nir"]
+
+
+def get_template_content(filename, ignore_context=False):
+    line_beginnings_to_ignore = ['"POT-Creation-Date']
+
+    if ignore_context:
+        line_beginnings_to_ignore += ["#:"]
+
+    with open(filename) as file:
+        return "".join(
+            filter(
+                lambda l: all(
+                    not l.startswith(param) for param in line_beginnings_to_ignore
+                ),
+                file.readlines(),
+            )
+        )
+
+
+def print_filename_results(filename, success=True):
+    if success:
```

Comment:
Printing successes is just noise. I'd only be interested in those that fail.

Reply:
I disagree - we do this with eq-schema-validator tests, and it allows us to identify what's been processed. Not sure why we'd adopt a different approach here.

Reply:
Do you ever parse the successful output line by line (either with human eyeball or some kind of scripted approach) to ensure it matches your expectations? If not, then I'm still pretty sure it's noise. It's far easier to grep the output if it is done on a per exception basis.

Reply:
Yes. Every time we inspect the output of a test run, or lint run we're doing exactly that. This is how we know a process is doing something and not just hanging. I think changing how scripts work to only output errors is outside the scope of this PR.